lib/resque.rb

```diff
@@ -165,6 +165,12 @@ def prune_interval
     @prune_interval || DEFAULT_PRUNE_INTERVAL
   end
 
+  attr_writer :enqueue_front
+  def enqueue_front
+    return @enqueue_front unless @enqueue_front.nil?
+    @enqueue_front = false
+  end
```

Comment:
I was just following the format of the existing code (https://github.com/resque/resque/pull/1425/files#diff-bb00e7d239ca836ced064c1f97814e23L159)

I guess this also allows setting a default value? If there is a better way, I'm down to implement that?